src/config_effects.c

```diff
@@ -33,7 +33,7 @@
 extern "C" {
 #endif
 /******************************************************************************/
-const char keeper_effects_file[]="effects.cfg";
+const char * keeper_effects_file = "effects.cfg";
```

Comment:
only effects.cfg gives a warning? all the other cfg's look like they're defined in the same way

Reply:
No, many of them did. Like _DK_imp_spangle_effects in my original comment. The differences between const char[] and const char * in this context is practically zero but generally people are more familiar with cons char * for string constants.